docs/topics_5/plugin-create-config.adoc

```diff
@@ -1,14 +1,14 @@
 // Module included in the following assemblies:
 // * docs/plugin-guide_5/master.adoc
 [id='create_run_config_{context}']
-= Creating an {ProductShortName} Run Configuration
+= Creating an {ProductShortName} run configuration
 
 {ProductShortName} run configurations can be created using the {ProductShortName} button (image:windup.png[{ProductShortName} button]). A run configuration specifies the project to analyze, migration path, and additional options for the execution. You can create multiple run configurations, and each must have a unique name.
 
 [discrete]
 == Input
 
-Migration Path::
+Migration path::
 Select a migration path, which determines which {ProductShortName} rulesets are used. The migration path defaults to *Anything to EAP 7*, but can be changed to any supported migration path.
```

Comment:
This has to be checked against screenshots

Reply:
There was no available screenshot. In context, it seems likely to be a UI label, so I reverted it.